lib/event/index.test.js

```diff
@@ -1,7 +1,6 @@
 "use strict";
 
 var assert = require("@sinonjs/referee").assert;
-var extend = require("just-extend");
```

Comment:
The `object-assign` ~~polyfill~~ ponyfill must be loaded for this to not break compatibility in IE11, otherwise it'll fail with `undefined is not a function`.

Reply:
Ups, I actually hadn't meant to include the `object-assign` ponyfill. That was an earlier experiment.

I wasn't aware sinon still supported IE 11 - my bad.

I guess that leaves two options for now
- do nothing and I'll just close this PR (which is completely fine IMO)
- switch to the `object-assign` ponyfill

The latter has two small advantages:
- it's a much more popular library so it's more likely to already exist in a project and be deduplicated. This is especially true in the react ecosystem as it's a dependency of react itself
- it makes sure (future )usage is 100% inline with the `Object.assign` spec making the switch to the real thing even moe trivial

Either option is completely fine with me but I'd appreciate some guidance before making changes to the PR.

Reply:
My thinking is that if it works, don't spend time fixing it. I appreciate the efforts, but could we just close this, given that it buys us/everyone else very little? Thinking that that time is better spent on the actual Object.assign task once IE11 is dropped.

I do appreciate the effort to make things better, just so that has been said :smile:

Reply:
All good. Thanks for the feedback :)